src/redux/reducer.js

```diff
@@ -106,6 +108,11 @@ function fetchFeatures() {
 
 function loadFeatures() {
   return function (dispatch) {
+    if (IS_TEST) {
+      dispatch({type: "FEATURES", payload: JSON.parse(loadGeoJSON("./src/data/" + geojsonURL)).features}); 
+      return;
+    }
+
```

Comment:
Shouldn't this not be necessary if the mock is working?

Reply:
Yeah the rebase got a bit messy and this must have slipped in